ext/posix/posix.stub.php

```diff
@@ -92,16 +92,16 @@ function posix_getrlimit(): array|false {}
 #endif
 
 #ifdef HAVE_SETRLIMIT
-function posix_setrlimit(int $resource, int $softlimit, int $hardlimit): bool {}
+function posix_setrlimit(int $resource, int $soft_limit, int $hard_limit): bool {}
 #endif
 
 function posix_get_last_error(): int {}
 
 /** @alias posix_get_last_error */
 function posix_errno(): int {}
 
-function posix_strerror(int $errno): string {}
+function posix_strerror(int $error_code): string {}
 
 #ifdef HAVE_INITGROUPS
-function posix_initgroups(string $name, int $base_group_id): bool {}
+function posix_initgroups(string $username, int $base_group_id): bool {}
```

Comment:
Does `$base_group_id` really require the `$base` prefix? What about just `$gid` to make it consistent with the rest of the parameters?

Reply:
Man page says:

>        The  initgroups() function initializes the group access list by reading the group database
>       /etc/group and using all groups of which user is a member.  The additional group group  is
>       also added to the list.

So yeah, I have no idea what the `base_` prefix is even supposed to mean, I'll drop it.